music21/converter/subConverters.py

```diff
@@ -1143,6 +1144,22 @@ def parseFile(self, fp, number=None):
         rtf.close()
         romanTextTranslate.romanTextToStreamScore(rtHandler, self.stream)
 
+    def write(self, obj, fp=None):
```

Comment:
To match the other `write()` signatures in other writers, you can use:
```
def write(self, obj, fmt, fp=None, subformats=None, **keywords):  # pragma: no cover
```
and that should take care of the `TypeError`s I was seeing. MSC might chime in on whether you can help yourself to the `# pragma: no cover` like elsewhere.

Reply:
Thanks @jacobtylerwalls! Duly replicated and committed. I've taken the liberty of including the `# pragma: no cover` on the basis that it affects a lot of `converter.subConverter` and is perhaps one for a separate PR.